### PR TITLE
Initial Project Setup

### PR DESCRIPTION
**Pull request by dimamo5**
This PR adds a very basic configuration and file structure to the project. 

Currently, this project is split into 2 separate sub-projects:
- `velocidi-sample`: an Android application demonstrating the capabilities of the SDK
- `velocidi-sdk`: SDK codebase.

Please ignore the boilerplate code inside `test` and `androidTest` directories. It's there to help me setup Travis pipeline.

### CODE REVIEW

**Comment by dimamo5** [comment]
@duartepinto 	Which files?

**Comment by duartepinto** [comment]
> @duartepinto Which files?

nvm, confused `gradle/` with `.gradle/`

File: velocidi-sample/build.gradle [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+        test.java.srcDirs += 'src/test/kotlin'
+        androidTest.java.srcDirs += 'src/androidTest/kotlin'
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```
Comment (jcazevedo) [inline velocidi-sample/build.gradle]:
```suggestion
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

File: velocidi-sdk/publish.gradle [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                }
+            }
+        }
+    }
+}
+
+signing {
+    required { !isSnapshot && gradle.taskGraph.hasTask("uploadArchives") }
+    sign configurations.archives
+}
```
Comment (jcazevedo) [inline velocidi-sdk/publish.gradle]: Can you please add a newline here?

File: velocidi-sdk/src/test/kotlin/com/velocidi/ExampleUnitTest.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see [Testing documentation](http://d.android.com/tools/testing)
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, (2 + 2).toLong())
+    }
+}
```
Comment (jcazevedo) [inline velocidi-sdk/src/test/kotlin/com/velocidi/ExampleUnitTest.kt]: Can you please add a newline here?

File: velocidi-sample/build.gradle (hunk 1 above)
Comment (dimamo5) [reply velocidi-sample/build.gradle]: Fixed in bd9bace

**Pull request merged**